server/plugin/configuration.go

```diff
@@ -250,3 +250,20 @@ func generateSecret() (string, error) {
 
 	return s, nil
 }
+
+func (c *Configuration) getOrganizations() []string {
+	if c.GitHubOrg == "" {
+		return nil
+	}
```

Comment:
This seems error-prone to me versus returning an empty array. Iterating through the array even if it's `nil` apparently still works, so it shouldn't matter, but just thinking about nil pointers here